app/components/simple-editor.js

```diff
@@ -282,46 +352,21 @@ export default class SimpleEditorComponent extends Component {
   }
 
   setupDatePlugin({ nodes, userConfig, config, nodeViews }) {
-    if (!userConfig.date) {
-      config.date = {
-        placeholder: {
-          insertDate: this.intl.t('date-plugin.insert.date'),
-          insertDateTime: this.intl.t('date-plugin.insert.datetime'),
-        },
-        formats: [
-          {
-            label: 'Short Date',
-            key: 'short',
-            dateFormat: 'dd/MM/yy',
-            dateTimeFormat: 'dd/MM/yy HH:mm',
-          },
-          {
-            label: 'Long Date',
-            key: 'long',
-            dateFormat: 'EEEE dd MMMM yyyy',
-            dateTimeFormat: 'PPPPp',
-          },
-        ],
-        allowCustomFormat: true,
-      };
-    } else {
-      config.date = userConfig.date;
-    }
+    config.date = maybeMergeUserConfig(
+      defaultRdfaDatePluginConfig(this.intl.t.bind(this.intl)),
+      userConfig.date
+    );
+
     nodes.date = date(config.date);
     nodeViews.date = (controller) => dateView(this.config.date)(controller);
```

Comment:
`this.config` is never set I think?
I did a quick look and I don't think the `config` property is really used at all.

Either all your merge calls should also update `this.config`, or `this.config` should completely be removed.

Reply:
right my bad. This also has to be like his because it is a lambda.